examples/multistep_workflow/main.py

```diff
@@ -43,19 +44,21 @@ def _already_ran(entry_point_name, parameters, source_version, experiment_id=Non
             eprint(("Run matched, but is not FINISHED, so skipping "
                     "(run_id=%s, status=%s)") % (run_info.run_id, run_info.status))
             continue
-        if run_info.source_version != source_version:
+
+        if git_commit != tags.get(mlflow_tags.MLFLOW_GIT_COMMIT, None):
             eprint(("Run matched, but has a different source version, so skipping "
-                    "(found=%s, expected=%s)") % (run_info.source_version, source_version))
+                    "(found=%s, expected=%s)") % (run_info.source_version, git_commit))
```

Comment:
should it be `tags.get(mlflow_tags.MLFLOW_GIT_COMMIT, None)` instead of `run_info.source_version` ?

Reply:
Good catch! Yes it should.